### ISSUE TITLE
Support for Play-Json 2.6.x

### ISSUE THREAD

**Issue by apatzer** [xdotai/play-json-extensions#33]
Play Json is now at 2.6.0-M6 and about to be released. It also natively supports default values using a syntax like this:

```
implict val jsonFormat = Json.using[Json.WithDefaultValues].format[MyCaseClass]
```
However, there is still not support for 22+ fields.

Any anticipated date to support 2.6.x?

**Comment by apatzer** [comment]
A simple version to support Play-Json 2.6.x is ready at https://github.com/xdotai/play-json-extensions/pull/34

However, you may want to refactor to take advantage of the aforementioned case class with defaults macros now in 2.6.x

**Comment by meddulla** [comment]
+1 Am trying to integrate with sangria-graphql which uses a later Play-Json version and this originates the same error.

**Comment by AndrewZurn** [comment]
+1 on this as well. Might be nice to have #34 merged into a seperate branch and a milestone release made so we can start using it for our 2.6 migrations.

**Comment by AndrewZurn** [comment]
Probably can be closed now given support for 2.6 and Scala 2.12 was just added.

**Comment by cvogt** [comment]
yeah, thx!

**Comment by cvogt** [comment]
cross released for 2.11 and 2.12 as version 0.10.0
